packages/react-native-web/src/vendor/react-native/Animated/nodes/AnimatedInterpolation/index.js

```diff
@@ -167,9 +167,12 @@ function interpolate(
 
 function colorToRgba(input: string): string {
   let normalizedColor = normalizeColor(input);
-  if (normalizedColor === null || typeof normalizedColor !== 'number') {
+  if (normalizedColor == null) {
     return input;
   }
+  if (typeof normalizedColor !== 'number') {
+    return normalizedColor;
+  }
 
   normalizedColor = normalizedColor || 0;
```

Comment:
I would suggest to remove the rest of this function as that logic won't be executed anymore as far as I can see. The same logic is implemented in `normalizecolor()`

This is how I would suggest the function to be instead. 
```js
function colorToRgba(input: string): string {
  const normalizedColor = normalizeColor(input);
  return normalizedColor ?? input; 
}
```

Reply:
Could still keep the check for normalizedColor !== 'number' if we wanted though.

Reply:
I'd like to avoid changing the vendor code logic if at all possible. It's copied straight from react-native. It might be that the `normalizedColor` function is not equivalent and the logic needs adjusted

Reply:
Understandable. The origin of the error seems to be the change from `normalize-css-color` to `modules/normalizeColor` 
The implementation of normalizeColor in modules will return the rgba string instead of the number value of the color. 

I tried copying the [interpolation tests from the react native](https://github.com/facebook/react-native/blob/master/Libraries/Animated/__tests__/Interpolation-test.js) repo and running them with normalize-css-color passes all tests, while the modules/normalizeColor breaks as it returns string. 

Changing the implementation of modules/normalizeColor breaks a lot of tests